irctest/server_tests/test_regressions.py

```diff
@@ -101,7 +105,7 @@ def testEmptyNick(self):
         self.addClient(1)
         self.sendLine(1, 'NICK :')
         self.sendLine(1, 'USER u s e r')
-        replies = set(msg.command for msg in self.getMessages(1))
+        replies = set(msg.command for msg in self.getMessages(1, synchronize=False))
```

Comment:
Seems as though this can pass spuriously if the server sends a NOTICE?

Reply:
indeed